.github/workflows/ci.yml

```diff
@@ -0,0 +1,66 @@
+name: "CI"
+
+#  not sure what to put in here
+env:
+  TS_PROFILE: "ci"
+
+on:
+  pull_request:
+    branches:
+      - "main"
+      - "develop"
+  push:
+    branches:
+      - "main"
+      - "develop"
+
+jobs:
```

Comment:
You added this job to check for the linked issue in the pull request description but didn't add an issue in the PR description 🙂. This action seems to be flakey, so for now, remove it.

Reply:
I don't think it's flakey, it's failing